lib/nodejs/worker.ex

```diff
@@ -149,9 +156,16 @@ defmodule NodeJS.Worker do
     end
   end
 
+  # Safely resets the terminal, handling potential errors if
+  # the port is already closed or invalid
   defp reset_terminal(port) do
-    Port.command(port, "\x1b[0m\x1b[?7h\x1b[?25h\x1b[H\x1b[2J")
-    Port.command(port, "\x1b[!p\x1b[?47l")
+    try do
```

Comment:
Added try/rescue to fix crashes when ports are closed during termination. Instead of blowing up with ArgumentError, we now just log and move on. This helps during dev reloading or when containers are shutting down - basically anytime processes are killed unpredictably.